tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -605,9 +608,16 @@ async def test_nft_with_did_wallet_creation(two_wallet_nodes: Any, trusted: Any)
     )
     assert resp.get("success")
     sb = resp["spend_bundle"]
-
+    memos = compute_memos(sb)
+    assert memos
+    puzhashes = []
+    for x in memos.values():
+        puzhashes.extend(list(x))
+    assert len(puzhashes) > 0
+    for puzhash in puzhashes:
+        assert wallet_0.wallet_state_manager.puzzle_store.get_derivation_record_for_puzzle_hash(puzhash)
```

Comment:
this works when non-synthetic pubkey is used too, so we need a different way to verify minted NFT will resync correctly